fe-nc-news-project/src/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom"
+import { fetchCommentsByArticleId } from "./utils";
+
+export default function Comments() {
+
+    const [comments, setComments] = useState([])
+    const {article_id} = useParams()
+    
+
+useEffect(() => {
+    fetchCommentsByArticleId(article_id).then((result) => {
+        setComments(result.data.comments)
+        console.log(comments);
+    })
+}, [])
+
+return (<>
+  <h4>Add Comment:</h4>
+  <input></input>
+  <h3>Comments</h3>
+  {comments.map((comment) => {
```

Comment:
We also want to add something, so that if there are no comments on an article yet, we display a message such as 'no comments yet'